sdk/oracle-service/tests/mocks/MockOracleProvider.ts

```diff
@@ -8,9 +9,18 @@ import {
   SpotPriceInfo,
 } from '@summerfi/sdk-common'
 import { IToken } from '@summerfi/sdk-common/common'
+import { ManagerProviderBase } from '@summerfi/sdk-server-common'
 
-export class MockOracleProvider implements IOracleProvider {
-  type: OracleProviderType = OracleProviderType.OneInch
+export class MockOracleProvider
+  extends ManagerProviderBase<OracleProviderType>
+  implements IOracleProvider
+{
+  constructor() {
+    super({
+      configProvider: undefined as unknown as IConfigurationProvider,
+      type: OracleProviderType.OneInch,
+    })
+  }
 
   getSupportedChainIds(): ChainId[] {
     return [1, 4, 6]
```

Comment:
Don't we have enum with those id's? I'm aware that it wasn't added in this PR

Reply:
We have the `ChainFamiliesMap` where those IDs are defined, but not in an enum. In any case this is a Mock, not sure if we want to make it better. Let me know and I can change that.